lib/modules/manager/gitlabci-include/extract.spec.ts

```diff
@@ -29,6 +31,17 @@ describe('modules/manager/gitlabci-include/extract', () => {
       expect(res?.deps).toHaveLength(3);
     });
 
+    it('extracts multiple embedded include blocks', () => {
+      const res = extractPackageFile(yamlWithTriggerRef);
+      expect(res?.deps).toMatchSnapshot();
+      expect(res?.deps).toHaveLength(2);
```

Comment:
```suggestion
      expect(res?.deps).toHaveLength(2);
      expect(res?.deps).toMatchSnapshot([
      {
        depName: "mikebryant/include-source-example",
        currentValue: "master",
        datasource: "gitlab-tags",
      },
        depName: "mikebryant/include-source-example",
        currentValue: "1.0.0",
        datasource: "gitlab-tags",
      ]);
```
Use a partial matcher for snapshots or `toMatchObject()`

Reply:
When implementing these assertions I was following the established pattern in this file. I think it may be better to make this change in a follow up PR that updates all of the tests in this file.

Reply:
no, please follow our new best practices, the other tests are using outdated pattern and needs migrations

Reply:
updated assertion to use partial matcher for snapshots